packages/web-runtime/src/components/SidebarNav/SidebarNav.vue

```diff
@@ -128,9 +141,12 @@ export default defineComponent({
       { deep: true, immediate: true }
     )
 
-    return { highlighterAttrs, navItemRefs }
+    return { highlighterAttrs, navItemRefs, backendVersion, webVersion }
   },
   computed: {
+    sideBarToggle() {
+      return sideBarToggle
+    },
```

Comment:
What's this for, am I overseeing something? 🤔

Reply:
wow, I have no Idea, my IDE is inventing stuff